src/features/session_replay/aggregate/index.js

```diff
@@ -153,7 +149,7 @@ export class Aggregate extends AggregateBase {
       if (fullSample) this.mode = MODE.FULL // full mode has precedence over error mode
       else if (errorSample) this.mode = MODE.ERROR
       // If neither are selected, then don't record (early return)
-      return
+      else return
```

Comment:
Same difference

Reply:
Not exactly. The else is Basically saying if it's "off" then return (ie don't run the rest of the function)

If we always return it'll never run

Reply:
Not exactly. The else is Basically saying if it's "off" then return (ie don't run the rest of the function)

If we always return it'll never run

Reply:
Ah lol, I didn't see the rest of the code.